lua/onedark/theme.lua

```diff
@@ -366,8 +366,9 @@ function M.setup(config)
     NvimTreeIndentMarker = {fg = c.fg_gutter},
     NvimTreeImageFile = {fg = c.fg_sidebar},
     NvimTreeSymlink = {fg = c.purple},
-    NvimTreeFolderName = {fg = c.fg_light},
-    NvimTreeOpenedFolderName = {fg = c.fg_light, style = "bold"},
+    NvimTreeFolderName = {fg = c.blue},
+    NvimTreeOpenedFolderName = {fg = c.blue, style = "bold"},
```

Comment:
Remove these changes because It makes nvimtree look more complex when git highlight is enabled.

Reply:
What do you mean by git highlight? :)

Reply:
[nvim_tree_git_hl](https://github.com/kyazdani42/nvim-tree.lua#setup) option in nvimtree. 
![image](https://user-images.githubusercontent.com/24286590/144704049-8ad2c254-2d30-4a16-a11e-d5590b5684ca.png)